Makefile

```diff
@@ -20,7 +20,7 @@ OVERRIDE_IMAGE_NAME?=$(FARGATE_COLLECTOR_TEST_IMAGE)
 LDFLAGS=-w -X main.version=$(VERSION) -X main.commit=$(GIT_COMMIT)
 
 # Binary names
-BINARY_NAME=fargate_collector
+BINARY_NAME=tanzu-observability-collector-for-aws-fargate
 BINARY_LINUX=$(BINARY_NAME)_linux
 BINARY_DARWIN=$(BINARY_NAME)_darwin
```

Comment:
Is this constant getting used any where in the script?  Does fargate collector supports multiple flavors of collector binary?

Reply:
This helps in creating the binary for Mac os. 
It will be helpful for the Mac OS developers to create and test the binary on Mac OS.

We only use Linux binary to create the container.